demo/js/components/immutable-demo.js

```diff
@@ -639,7 +639,7 @@ class App extends React.Component {
                   <VictoryTooltip
                     y={150}
                     flyoutStyle={{ fill: "white" }}
-                    style={[{ fill: "green" }, { fill: "magenta" }]}
```

Comment:
`style` should actually work with objects or arrays of objects for `VictoryLabel` and `VictoryTooltip`

Reply:
ok, i will fix the `style` prop in `VictoryTooltipProps` interface to take into account the arrays of objects